### PR TITLE
PMM-5362 Adding tests for Remote Instances

### PR DESCRIPTION
**Pull request by IvanaWzientkova**

### CODE REVIEW

File: tests/pages/remoteInstancesPage.js [hunk 1]
```diff
@@ -37,6 +39,7 @@ module.exports = {
         usePerformanceSchema: "//input[@name='qan_mysql_perfschema']",
         usePerformanceSchema2: "//input[@name='qan_mysql_perfschema']/following-sibling::span[2]",
         skipTLSL: "//input[@name='tls_skip_verify']/following-sibling::span[2]",
+        skipConnectionCheck: "//*[@id='antd']/form/div[4]/label[1]/span[2]",
```
Comment (puneet0191) [inline tests/pages/remoteInstancesPage.js:42]: please don't use * in xpath, it will be more slower

File: tests/pages/remoteInstancesPage.js [hunk 2]
```diff
@@ -61,51 +63,84 @@ module.exports = {
...
-    fillRemoteMySQLFields(serviceName) {
+    fillRemoteFields(serviceName) {
+        switch(serviceName){
+            case 'mysql_remote_new':
+                I.fillField(this.fields.hostName, process.env.REMOTE_MYSQL_HOST);
+                I.fillField(this.fields.userName, process.env.REMOTE_MYSQL_USER);
+                I.fillField(this.fields.password, process.env.REMOTE_MYSQL_PASSWORD);
+                break;
+            case 'mongodb_remote_new':
+                I.fillField(this.fields.hostName, /*globalMongoHost*/);
```
Comment (puneet0191) [inline tests/pages/remoteInstancesPage.js:97]: Good, use variables just like above, `process.env.REMOTE_MONGO_HOST`

File: tests/pages/remoteInstancesPage.js [hunk 3]
```diff
@@ -37,6 +39,7 @@ module.exports = {
         usePerformanceSchema: "//input[@name='qan_mysql_perfschema']",
         usePerformanceSchema2: "//input[@name='qan_mysql_perfschema']/following-sibling::span[2]",
         skipTLSL: "//input[@name='tls_skip_verify']/following-sibling::span[2]",
+        skipConnectionCheck: "//[@id='antd']/form/div[4]/label[1]/span[2]",
```
Comment (yurkovychv) [inline tests/pages/remoteInstancesPage.js:42]: Can you use more stable selector here? as i see there is input with name="skip_connection_check" attr

Comment (IvanaWzientkova) [reply tests/pages/remoteInstancesPage.js:42]: Sure! :) I'll look at it :)

File: tests/verifyRemoteInstances_test.js [hunk 4]
```diff
@@ -27,15 +26,43 @@ xScenario('Verify is the remote instances are in Running Status @pmm-post-update
...
+
+xScenario('Open Remote Instance Page and Add ProxySQL instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
+    let proxysql_service_name = "proxysql_remote_new";
+    let version = "new";
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilNewRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('proxysql');
+    remoteInstancesPage.fillRemoteFields(proxysql_service_name);
+    remoteInstancesPage.createNewRemoteInstance(proxysql_service_name, version);
+    pmmInventoryPage.verifyMySQLRemoteServiceIsDisplayed(proxysql_service_name);
```
Comment (yurkovychv) [inline tests/verifyRemoteInstances_test.js:66]: could you please rename this method to more generic name?

File: tests/verifyRemoteInstances_test.js [hunk 5]
```diff
@@ -10,32 +10,57 @@ xScenario('Open Remote Instance Page and Add mysql instances @pmm-pre-update', a
...
 
+xScenario('Open Remote Instance Page and Add PostgreSQL instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
+    let postgresql_service_name = "postgresql_remote_new";
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilNewRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('postgresql');
+    //need to be add methods for create and verification after I got credentials
+});
+
+xScenario('Open Remote Instance Page and Add ProxySQL instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
```
Comment (yurkovychv) [inline tests/verifyRemoteInstances_test.js:56]: could you please rename these scenarios to one template like : Verify user is able to add ProxySQL instance ?

**Pull request merged**